ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1102,3 +1102,40 @@ def test_tensorflow_strided_slice(
         if hasattr(e, 'message'):
             if "only stride 1 allowed on non-range indexing" in e.message:
                 assume(False)
+
+
+# tile
+@handle_frontend_test(
+    fn_tree="tensorflow.tile",
+    dtypes_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")),
+    dt_and_multiples=helpers.dtype_and_values(
+        available_dtypes=["int32", "int64"]
```

Comment:
And like I've said in a previous comment, you need to control the shape of the produced `multiples` arrays. 
I would suggest to generate `dtypes_and_input` and `dt_and_multiples` in a `st.composite` helper function, where you get the shape of `input` and use it to define the number and size of dimensions of `multiples` using the corresponding `dtype_and_values` arguments.

Reply:
noted. In order to get the dimensions of ```input```, I've used ```len(ivy.shape)```. Would using the argument ```shape=True``` in the helper function be better in this case? I saw an example of it used previously for some of the test functions.

Reply:
I don't think one way is better than the other. You can just leave it as it is.

Reply:
I'm getting the error: ```F821 undefined name 'ivy'```

Reply:
For the next commit, I'll add ```ret_shape=True``` to where I defined ```dtypes_n_input```

Reply:
I think there's a mismatch in the number of values to unpack, and this leads to the following error: 
```ValueError: too many values to unpack (expected 2)```

```python
dtypes_n_input, input_dims = draw(
        helpers.dtype_and_values(
            available_dtypes=helpers.get_dtypes("numeric"),
            ret_shape=True
        ))
```

I'll try adding another argument to catch the input as well.

Reply:
@AnnaTz 

```hypothesis.errors.InvalidArgument: min_value=() must be a real number```

I think this error arises because I'm not able to get the right shape onto ```input_dims```. Is there an explanation on the ```helpers.dtype_and_values``` function I can refer to?

Reply:
I have made a comment trying to explain what you're doing wrong, but if you'd like more detail you can look into the `dtype_and_values` source code to see what values are returned.